ops/reth-cross-test/phases/base.py

```
@@ -0,0 +1,425 @@
+    """
+    genesis_path = config.DOCKER_DIR.parent / "l2-genesis" / ".devnet" / "genesis-l2.json"
+    env_path = config.DOCKER_DIR / ".env"
+
+    # Check if devnet is already initialized
+    l1_ready = False
+    try:
+        w3 = Web3(Web3.HTTPProvider(config.L1_RPC, request_kwargs={"timeout": 5}))
+        if w3.eth.block_number > 0:
+            l1_ready = True
+    except Exception:
+        pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)